src/lib/services/scheduler-service.test.ts

```diff
@@ -0,0 +1,122 @@
+import { LogProvider } from '../logger';
```

Comment:
Why is this file new?

Reply:
Not sure, I rebased the branch from main